File: APIServices/Conflux/Models/OTA/Rates/Rate.cs
using System;
using System.Collections.Generic;
using APIServices.Conflux.Enum;
namespace APIServices.Conflux.OTA.Models.Rates
{
    public class Rate
    {
        public bool IsPromotion { get; set; }
        public bool HasPriceException { get; set; }
        public TypeRateEnum TypeRate { get; set; }
        public string StartDate { get; set; }
        public string EndDate { get; set; }
        public List<BaseGuestAmount> BaseGuestAmounts { get; set; }
        public List<AdditionalGuestAmount> AdditionalGuestAmounts { get; set; }      
        public bool ApplyMon { get; set; }
        public bool ApplyTue { get; set; }
        public bool ApplyWed { get; set; }
        public bool ApplyThu { get; set; }
        public bool ApplyFri { get; set; }
        public bool ApplySat { get; set; }
        public bool ApplySun { get; set; }
    }
}

File: RateManagerCss/Models/OtaRequest/Rate.vb
Public Class Rate

    Private start As String
    Public Property StartDate() As String
        Get
            Return start
        End Get
        Set(ByVal value As String)
            start = value
        End Set
    End Property

    Private [end] As String
    Public Property EndDate() As String
        Get
            Return [end]
        End Get
        Set(ByVal value As String)
            [end] = value
        End Set
    End Property

    Private bseGuestAmountList As List(Of BaseGuestAmount)
    Public Property BaseGuestAmountsList() As List(Of BaseGuestAmount)
        Get
            Return bseGuestAmountList
        End Get
        Set(ByVal value As List(Of BaseGuestAmount))
            bseGuestAmountList = value
        End Set
    End Property

End Class
